lighthouse/helpers/reports.py

```diff
@@ -149,7 +149,7 @@ def map_labware_to_location(labware_barcodes):
     labware_to_location_barcode = [
         {
             "plate_barcode": record["barcode"],
-            "location_barcode": record["location"].get("barcode", ""),
+            "location_barcode": str(record["location_barcode"] or ""),
         }
```

Comment:
Not quite sure why there is a string conversion here (assuming that's what str() does), are some barcodes coming across the API as integers?

Reply:
This is a bit of guess work. Because we are returning the results as json if the labware has a null location I have set it to return the plate barcode as null. I think this should be the correct behaviour even though null location exists. Python does not recognise null and json.dumps will convert it to None. i think!. None is false so we convert it to an empty string. The other option would be to return an empty string from Labwhere. I did a fair bit of research but for such a popular language the online information seems poor compared to Ruby.